src/main/proto/BUILD

```diff
@@ -1,10 +1,14 @@
-load("@org_pubref_rules_protobuf//java:rules.bzl", "java_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_proto_grpc//java:defs.bzl", "java_grpc_library")
```

Comment:
Any reason not to use the rules that come with the grpc_java?

(similar to here: https://github.com/grpc-ecosystem/polyglot/blob/43e31743dccbde8e61f4735a723fc975af1c687e/src/main/proto/BUILD#L3)

Reply:
I've simply replaced the stackb rules with more up to date ones and didn't think about the grpc-java ones. Updated in new version